istio.yaml

```diff
@@ -0,0 +1,428 @@
+config:
+  log-level: debug
+  mesh: istio
+  istio:
+    prometheusUrl: http://prometheus.istio-system.svc.cluster.local:9090
+    workloadQueries:
+      resourceQueries:
+        p99_response_latency : |-
+          histogram_quantile(
+                            0.99,
+                            sum(
+                              irate(
+                                istio_request_duration_seconds_bucket{
+                                destination_owner=~"{{ printf "kubernetes://apis/.+/namespaces/%s/%ss/%s" (default ".+" .namespace) (default ".+" (lower .kind)) (default ".+" .name) }}",
+                                }[{{.window}}]
+                              )
+                            ) by (
+                              destination_owner,
+                              le
+                            )
+                          )
+        p90_response_latency : |-
+          histogram_quantile(
+                            0.90,
+                            sum(
+                              irate(
+                                istio_request_duration_seconds_bucket{
+                                destination_owner=~"{{ printf "kubernetes://apis/.+/namespaces/%s/%ss/%s" (default ".+" .namespace) (default ".+" (lower .kind)) (default ".+" .name) }}",
+                                }[{{.window}}]
+                              )
+                            ) by (
+                              destination_owner,
+                              le
+                            )
+                          )
+        p50_response_latency : |-
+          histogram_quantile(
+                            0.50,
+                            sum(
+                              irate(
+                                istio_request_duration_seconds_bucket{
+                                destination_owner=~"{{ printf "kubernetes://apis/.+/namespaces/%s/%ss/%s" (default ".+" .namespace) (default ".+" (lower .kind)) (default ".+" .name) }}",
+                                }[{{.window}}]
+                              )
+                            ) by (
+                              destination_owner,
+                              le
+                            )
+                          )
+        success_count : |-
```

Comment:
Hmmm, But if we are going to do the templating queries as per. making these into single query strings wouldn't be useful. 
If you think it will be useful, will do it.

Reply:
Let's handle all the query reduction work in a separate PR. I think there's some low hanging fruit to make it a little less naive.